tests/adapter/dbt/tests/adapter/basic/test_docs_generate.py

```diff
@@ -404,6 +404,22 @@ def project_config_update(self, unique_schema):
             },
         }
 
+    @pytest.fixture(autouse=True)
```

Comment:
Looks like this might have broken the docs tests? https://github.com/dbt-labs/dbt-core/actions/runs/7065010846/job/19307239764?pr=9195#step:8:432

Could be an inter-test dependency (i.e. docs generate was relying on some artifacts _not_ being cleaned up)

Reply:
going to try and swap over to specifying `unqiue_schema` instead of `test_schema` in clean_up method and see if that fixes it `test_schema` should ideally be being picked up by the default teardown method we have designed anyway.